lib/swaggerValidator/modelValidator.ts

```diff
@@ -624,7 +624,7 @@ export class SwaggerExampleValidator {
           node = jsonPointer.get(this.swagger, jsonRef);
         } else {
           const externalSwagger = this.jsonLoader.getFileContentFromCache(err.source.url);
-          node = jsonPointer.get(externalSwagger!, jsonRef);
+          node = jsonPointer.get(externalSwagger as openapiToolsCommon.JsonObject, jsonRef);
```

Comment:
Why change this? Does it fix lint error?

Reply:
It's to fix compile error, not sure why it could pass before
```
lib/swaggerValidator/modelValidator.ts:627:34 - error TS2345: Argument of type 'string | number | boolean | {} | (JsonObjectInterface & object) | JsonArray' is not assignable to parameter of type 'JsonObject'.
  Type 'string' is not assignable to type 'JsonObject'.

627           node = jsonPointer.get(externalSwagger!, jsonRef);
```